pkgs/tools/nix/nixos-render-docs/src/nixos_render_docs/manual.py

```diff
@@ -238,25 +239,38 @@ class HTMLParameters(NamedTuple):
     toc_depth: int
     chunk_toc_depth: int
     section_toc_depth: int
+    media_dir: Path
 
 class ManualHTMLRenderer(RendererMixin, HTMLRenderer):
     _base_path: Path
+    _in_dir: Path
     _html_params: HTMLParameters
 
     def __init__(self, toplevel_tag: str, revision: str, html_params: HTMLParameters,
                  manpage_urls: Mapping[str, str], xref_targets: dict[str, XrefTarget],
-                 base_path: Path):
+                 in_dir: Path, base_path: Path):
         super().__init__(toplevel_tag, revision, manpage_urls, xref_targets)
-        self._base_path, self._html_params = base_path, html_params
+        self._in_dir, self._base_path, self._html_params = in_dir, base_path.absolute(), html_params
```

Comment:
Do we need to save lines like this? It is obscures diffs and the non-locality makes the assignments harder to reason about.